src/components/Chat/InputMessageV2.tsx

```diff
@@ -154,7 +154,7 @@ const InputMessageV2 = ({
       video_name: media?.nameVideo ?? null,
       video_type: media?.typeVideo ?? null
     }));
-    onSendButtonClicked('Sent media 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent media 🏞️', resultUrls);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the message string from 'Sent media 🎆🌆🌉' to 'Sent media 🏞️' doesn't seem to have any functional impact. However, it's important to ensure that these changes are consistent across all instances where this message is used to maintain a uniform user experience.
```diff
-    onSendButtonClicked('Sent media 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent media 🏞️', resultUrls);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->